### PR TITLE
Add page about required management steps

### PR DESCRIPTION
**Pull request by huffa**
Created a new page that describes how to run the space.

### CODE REVIEW

File: src/SUMMARY.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
 - [Onboarding New Members](onboarding.md)
 - [Liability Waivers](waivers.md)
 - [Updating this Manual](editing.md)
-
+- [Management Responsibilities](leagal_management.md)
```
Comment (kryptographik) [inline src/SUMMARY.md]: Typo in legal

File: src/legal_management.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+### Secretary of State annual report
+* Who: Current Secretary
+* When: June 30
+* What: Annual Report can be filed at https://www.sos.wa.gov/corporations-charities (Click “Express Annual Report”)
+* Cost: $60
+
+### City of Seattle requirements
+* We don’t file anything with the city, but possibly we’re supposed to?
+* Michael was going to look into this for us, not sure what he found out
```
Comment (kryptographik) [inline src/legal_management.md]: No need to throw michael under the bus on the wiki :p

I reached out to the city and set up an appointment for next tuesday to figure this out, so we can just leave this section out for now and update it in a week

**Review by kryptographik — APPROVED** [review]

File: src/legal_management.md (hunk 2 above)
Comment (huffa) [reply src/legal_management.md]: Yeah, mostly didn't want to loose context. Also I was hoping he'd review and comment if he'd figured it out :)

Comment (michaelkirk) [reply src/legal_management.md]: I deserve to be thrown under that bus, but good looking out Noah. =)

**Pull request merged**